tuples/src/main/java/org/web3j/tuples/generated/Tuple10.java

```diff
@@ -48,42 +48,82 @@ public T1 getValue1() {
         return value1;
     }
 
+    public T1 component1() {
+        return value1;
+    }
+
     public T2 getValue2() {
```

Comment:
It is worth modifying the code generator to make these methods @Deprecated and move across to the `componentN` convention for web3j 5.0

Reply:
Addressed, please see the last commit.